cloudhub-2/modules/ROOT/pages/ch2-configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,60 @@
+= Configuring Horizontal Autoscaling (HPA) for CH2 Deployments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can configure horizontal scaling of Mule applications to make them responsive to resource usage by automatically scaling up or down the deployment replicas as needed.
+
+In Kubernetes, a Horizontal Pod Autoscaler (HPA) automatically updates a workload resource, with the aim of automatically scaling the workload to match demand. Horizontal scaling means that the response to increased load is to deploy more pods. For more information, visit the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale[Kubernetes documentaton^]. 
+
+== Before You Begin
+
+* Enable the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis[support for metrics APIs^] on your managed K8s API server.
++
+Refer to your managed vendors' documentation on how to install or enable metrics API servers.
+ 
+== Configure Horizontal Autoscaling
+
+To configure horizontal autoscaling:
+
+. From Anypoint Platform, select *Runtime Manager* > *Applications*.
+. Click *Deploy application*.
+. In the *Runtime* tab, check the *Enable Autoscaling* box.
+. Set the minimum and maximum *Replica Count* limits.
+. Click *Deploy Application*.
+
+image::ch2-config-autoscaling.png[Configure horizontal autoscaling.]
+
+
+== Limitations
+
+* Horizontal autoscaling does not work with clustering and rate limiting.
+* Horizontal autoscaling works with 0.1 vCcore replica size only. You cannot reserve or limit CPU resources. 
+
+
+== Autoscaling Status and Logs
+
+When an autoscaling event occurs and your Mule application with horizontal autoscaling scales up, you can check the *Scaling* status by clicking *View status* in your application's details window:
```

Comment:
Pl add: You can also see the *Scaling* status against your application in the Applications List in Runtime Manager.